### DISCUSSION TITLE
What composition do you use?

### DISCUSSION THREAD

**Discussion by Jokiblack** [TeamFightTacticsBots/Alune#57] [General]
Good evening,

I'm testing witchers and bastion, I get almost 50% win rate, what is your win rate and what composition do you use?

**Comment by Jokiblack** [comment]
![image](https://github.com/user-attachments/assets/4d518cfc-dbfe-4037-92be-1fd8b505cf8f)
my tests

**Reply by StereoSayan** [reply]
You can have the bot use different compositions?

**Reply by Jokiblack** [reply]
Yes, of course, in the configuration you can set it however you want, changing the traits, I have it like this
![image](https://github.com/user-attachments/assets/1060ba9e-ed4f-4a45-955f-ce88cdd2878a)
Go alune-output/config

**Reply by StereoSayan** [reply]
And how do you check your time played, win/loss ratio and winrate?

**Reply by Rafinhi** [reply]
@Jokiblack I am wondering, what would you suggest for set 13? I have created a few different combinations and they all seem to do not very well (with average placement of around 7). I am wondering if this set is just hard, or my MMR is too high for now.

**Reply by slimegirls** [reply]
The default config (Rebel + Sentinel) seems to be good due to having 2 1-costs and 2 2-costs in the vertical trait, and Rebel has 2 Sentinels already (Irelia and Illaoi). I've gotten a few top 4s with it, so maybe your MMR isn't low enough yet.

I was thinking Family could be good, but you need all 3 on the board which is heavily RNG dependent depending on how fast you get all 3 of them.

**Reply by Rafinhi** [reply]
I have tried Bruiser + Enforcer and the results are very mixed (sometimes it seems to get godlike rolls, and sometimes bench gets populated with random units too fast), Ill go back to Rebel + Sentinel to see if there is any change in regard to my previous results 🥰

**Comment by Jokiblack** [comment]
![image](https://github.com/user-attachments/assets/aeb45d3e-1a6e-425f-8cdd-333f56c42d9f)
I got a 50% win rate

**Reply by slimegirls** [reply]
Wow do you just have Black Rose in? I imagine if you can hit it early its an easy snowball with Sion against weak boards.

**Reply by Jokiblack** [reply]
> Wow do you just have Black Rose in? I imagine if you can hit it early its an easy snowball with Sion against weak boards.

I don't know why but it works haha

**Reply by Rafinhi** [reply]
Thank you, I have started testing (for anyone wondering, you need to use the python version as far as I see the released exe doesn't work properly since name of the black rose trait file is wrong)

**Reply by akshualy** [reply]
> Thank you, I have started testing (for anyone wondering, you need to use the python version as far as I see the released exe doesn't work properly since name of the black rose trait file is wrong)

No, the exe has been updated. I just didn't trigger a version bump to fix the trait name.

**Reply by Jokiblack** [reply]
![image](https://github.com/user-attachments/assets/0075a46b-d66a-4c5e-9091-56850419aca9)
I had some internet problems and lost 2 games because of it, but that's how the statistics go, it's brutal, that I keep 50%

**Reply by StereoSayan** [reply]
Could you show what your config looks like?
